src/extension.ts

```diff
@@ -482,10 +500,8 @@ export class Extension implements RunHooks {
           return;
 
         const trace = result.attachments.find(a => a.name === 'trace')?.path || '';
-        // if trace viewer is currently displaying the trace file about to be replaced, it needs to be refreshed
-        const prevTrace = (testItem as any)[traceUrlSymbol];
         (testItem as any)[traceUrlSymbol] = trace;
-        if (enqueuedSingleTest || prevTrace === this._traceViewer.currentFile())
+        if (enqueuedSingleTest)
```

Comment:
Can we land this refactoring separately?

Reply:
Actually, that was a mistake of mine, I'm going to rollback it.